pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -154,6 +155,13 @@ func changefeedPlanHook(
 			return err
 		}
 
+		for key, value := range opts {
```

Comment:
Could you also add some tests for this? Maybe add to `changefeed_test.go` ([TestChangefeedErrors](https://github.com/cockroachdb/cockroach/blob/master/pkg/ccl/changefeedccl/changefeed_test.go#L2451))